pandas/core/algorithms.py

```diff
@@ -910,8 +910,8 @@ def __init__(self, obj, n, keep):
         self.n = n
         self.keep = keep
 
-        if self.keep not in ('first', 'last'):
```

Comment:
need to add back to the docs-string (don't need a version added as it was tehre before)